react-native-sdk/update_dependencies.js

```diff
@@ -21,6 +24,13 @@ function updateDependencies() {
             packageJSON.dependencies[key] = RNSDKpackageJSON.peerDependencies[key];
             updated = true;
         }
+
+        if (semver.satisfies(RNSDKpackageJSON.peerDependencies[key], `> ${packageJSON.dependencies[key]}`)) {
```

Comment:
If it doesn't saisfy the requirement we need do something about it: log something, tell the user...